### PR TITLE
SO-5690: Add missing fields to SnomedComponent builders

### PR DESCRIPTION
**Pull request by AAAlinaaa**

### CODE REVIEW

**Review by cmark — CHANGES_REQUESTED** [review]
🥓 
Could you please add a test case that breaks without these? Thanks!

File: snomed/com.b2international.snowowl.snomed.core.rest.tests/src/com/b2international/snowowl/snomed/core/rest/versioning/SnomedVersioningApiTest.java [hunk 1]
```diff
@@ -99,6 +105,32 @@ public void forceCreateVersionShouldUpdateEffectiveTime() {
...
+				"moduleId", Concepts.MODULE_SCT_CORE,
+				"referenceSetId", Concepts.REFSET_CONCEPT_INACTIVITY_INDICATOR,
+				"referencedComponentId", conceptId,
+				"valueId", Concepts.PENDING_MOVE
+			));
+		
+		SnomedConcepts conceptBeforeVersioning = searchConcept(new CodeSystemURI(INT_CODESYSTEM), Map.of(
+				"activeMemberOf", REFSET_CONCEPT_INACTIVITY_INDICATOR,
+				"id", conceptId), 1);
+		assertTrue(conceptBeforeVersioning.getTotal() == 1);
```
Comment (cmark) [inline snomed/com.b2international.snowowl.snomed.core.rest.tests/src/com/b2international/snowowl/snomed/core/rest/versioning/SnomedVersioningApiTest.java]: Use `assertThat(conceptBeforeVersioning.getTotal()).isEqualTo(1)` instead.

File: snomed/com.b2international.snowowl.snomed.core.rest.tests/src/com/b2international/snowowl/snomed/core/rest/versioning/SnomedVersioningApiTest.java [hunk 2]
```diff
@@ -99,6 +105,32 @@ public void forceCreateVersionShouldUpdateEffectiveTime() {
...
+		
+		createVersion(INT_CODESYSTEM, "versionToTestDocumentPreservation", getNextAvailableEffectiveDateAsString(INT_CODESYSTEM)).statusCode(201);
+		getVersion(INT_CODESYSTEM, "versionToTestDocumentPreservation").statusCode(200);
+		
+		SnomedConcepts conceptAfterVersioning = searchConcept(new CodeSystemURI(INT_CODESYSTEM), Map.of(
+				"activeMemberOf", REFSET_CONCEPT_INACTIVITY_INDICATOR,
+				"id", conceptId), 1);
+		assertTrue(conceptAfterVersioning.getTotal() == 1);
```
Comment (cmark) [inline snomed/com.b2international.snowowl.snomed.core.rest.tests/src/com/b2international/snowowl/snomed/core/rest/versioning/SnomedVersioningApiTest.java]: And here as well.

File: snomed/com.b2international.snowowl.snomed.core.rest.tests/src/com/b2international/snowowl/snomed/core/rest/versioning/SnomedVersioningApiTest.java [hunk 3]
```diff
@@ -99,6 +105,32 @@ public void forceCreateVersionShouldUpdateEffectiveTime() {
 		assertEquals(versionEffectiveDate, EffectiveTimes.format(afterForceVersioning.getEffectiveTime(), DateFormats.SHORT));
 	}
 	
+	@Test
+	public void createVersionShouldPreserveDocumentPRoperties() {
+		String conceptId = createConcept(new CodeSystemURI(INT_CODESYSTEM), SnomedRestFixtures.childUnderRootWithDefaults());
```
Comment (cmark) [inline snomed/com.b2international.snowowl.snomed.core.rest.tests/src/com/b2international/snowowl/snomed/core/rest/versioning/SnomedVersioningApiTest.java]: Extract the URI to a local variable.

**Review by cmark — CHANGES_REQUESTED** [review]

**Review by cmark — APPROVED** [review]
LGTM!

**Comment by apeteri** [comment]
https://github.com/b2ihealthcare/snow-owl/pull/905 adds the missing builder method calls to concept documents, but the rest still needs to be ported to 8.x.

**Pull request merged**